src/Extensions/YiiTwigExtension.php

```diff
@@ -31,6 +31,70 @@ public function getFunctions(): array
                 /** @return mixed */
                 fn (string $id) => $this->container->get($id),
             ),
+            new TwigFunction(
```

Comment:
Seems, we can split to several extensions (`ContainerExtension`, `ArrayHelperExtension`, ...)

Reply:
And I think it would be good to make these changes in separate PRs.